docs/modules/operation/pages/deep-dive/user-management/security-roles.adoc

```diff
@@ -2,96 +2,102 @@
+| Permissions only to view the dashboard.
 
 | ROLE_DELEGATE
-| Allow actions (such as acknowledging an alarm) to be performed on behalf of another user.
+| Permissions to perform actions (such as acknowledging an alarm) on behalf of another user.
 
 | ROLE_DEVICE_CONFIG_BACKUP
-| Allow user to view and trigger device configuration backups.
+| Permissions to view and trigger device configuration backups.
 
 | ROLE_FLOW_MANAGER
-| Allow user to edit flow classifications.
+| Permissions to edit flow classifications.
 
 | ROLE_JMX
-| Allow retrieving JMX metrics but do not allow executing MBeans of the {page-component-title} JVM, even if they just return simple values.
+| Permissions to retrieve JMX metrics but does not not to execute MBeans of the {page-component-title} JVM, even if they just return simple values.
 ifndef::opennms-prime[]
 
 | ROLE_MINION
 | Minimum required permissions for a Minion to operate.
 endif::opennms-prime[]
 
 | ROLE_MOBILE
-| Allow user to use OpenNMS COMPASS mobile application to acknowledge alarms and notifications via the REST API.
+| Permissions to use OpenNMS COMPASS mobile application to acknowledge alarms and notifications via the REST API.
 
 | ROLE_PROVISION
-| Allow user to use the provisioning system and configure SNMP in {page-component-title} to access management information from devices.
+| Permissions to use the provisioning system and configure SNMP in {page-component-title} to access management information from devices.
 
-| ROLE_READONLY*
-| User limited to reading information in the web UI; unable to change alarm states or notifications.
+| ROLE_READONLY ^*^
+| Permissions only to read information in the web UI; user cannot change alarm states or notifications.
 
 | ROLE_REPORT_DESIGNER
 | Permissions to manage reports in the web UI and REST API.
 
 | ROLE_REST
-| Allow users to interact with the entire {page-component-title} REST API.
+| Permissions to interact with the entire {page-component-title} REST API.
 
-| ROLE_RTC*
-| Exchange information with the {page-component-title} Real-Time Console for availability calculations.
+| ROLE_RTC ^*^
+| Permissions to exchange information with the {page-component-title} Real-Time Console for availability calculations.
 
-| ROLE_USER*
+| ROLE_USER ^*^
 | Default permissions for a new user to interact with the web UI: can escalate and acknowledge alarms and notifications.
 |===
 
-== Assign roles to users
+== Assign security roles
+
+Follow the steps below to assign security roles to user accounts:
 
-. Log in as a user with admin permissions.
-. Click the gear icon at the top-right of the screen.
-. Choose menu:Configure OpenNMS[Configure Users, Groups and On-Call Roles > Configure Users].
-. Click the modify icon next to the user you want to update.
-. Select the role from menu:Security Roles[Available Roles].
-. Click *Add* to assign the security role to the user.
+. Click the *Gear* symbol at the top-right of the screen.
```

Comment:
We can leave this for now, until we decide to formally go back to not bolding symbols. :-)